src/renderer/components/views/projects/ProjectsView.tsx

```diff
@@ -0,0 +1,230 @@
+      .filter((item) => item.tonsCo2 > 0 && (item.methodology || item.projectType))
+      .sort((a, b) => (b.tonsCo2 || 0) - (a.tonsCo2 || 0))
+      .slice(0, 3);
+
+    const labels = filtered.map((item) => {
+      const label = item.methodology || item.projectType;
+      return label ? (label.length > 35 ? label.slice(0, 35) + '...' : label) : '';
+    });
+
+    const chartData = filtered.map((item) => item.tonsCo2);
+
+    return createChartData(labels, chartData, 'tonsCo2', COLORS.BAR);
+  };
+  const selfHostedCount = projectsHostedCountData.selfHostedProjectCount;
+  const externallyHostedCount = projectsHostedCountData.externallyHostedProjectCount;
+
+  const createChartData = (labels: string[], data: number[], label: string, backgroundColor: string[]) => ({
+    labels,
+    datasets: [
+      {
+        label,
+        data,
+        backgroundColor,
+        borderWidth: 1,
+      },
+    ],
+  });
+
+  const charts = [
+    {
+      data: generateBarChartData(issuedCarbonByMethodologyData.issuedTonsCo2),
+      title: 'Issued Carbon by Methodology',
+      type: 'bar',
+    },
+    {
+      data: generateBarChartData(issuedCarbonByProjectTypeData.issuedTonsCo2),
+      title: 'Issued Carbon by Project Type',
+      type: 'bar',
+    },
+    {
+      data: createChartData(
+        ['Externally Hosted Projects', 'Self Hosted Projects'],
+        [externallyHostedCount, selfHostedCount],
+        'Count',
+        COLORS.PIE,
+      ),
+      type: 'pie',
+    },
+  ];
+  if (
+    projectsStatusCountLoading ||
+    projectsHostedCountLoading ||
+    issuedCarbonByMethodologyLoading ||
+    issuedCarbonByProjectTypeLoading
+  ) {
+    return null;
+  }
+
+  return (
+    <div className="flex flex-col gap-16 m-4">
+      <div className="grid grid-cols-1 gap-4 md:grid-cols-3">
+        {filteredData?.map((status) => (
+          <Card key={status.projectStatus}>
+            <div className="flex flex-col gap-4 p-4 text-center">
+              <h3 className="text-7xl bold">{status.count}</h3>
+              <h4 className="text-lg font-semibold">
+                <a
+                  href="#glossary"
+                  onClick={() => openModal()}
+                  className={`${
+                    status.projectStatus.toLowerCase() === 'approved'
+                      ? 'text-lime-500 text-left dark:text-green-400'
+                      : status.projectStatus.toLowerCase() === 'authorized'
+                        ? 'text-left text-sky-400 dark:text-blue-500'
+                        : 'text-sky-800 dark:text-white'
+                  } underline`}
+                >
+                  Projects {status.projectStatus}
+                </a>
+              </h4>
+            </div>
+          </Card>
+        ))}
+        <GlossaryModal onClose={() => setActive(false)} open={active} />
+      </div>
+      <div className="grid grid-cols-1 gap-4 md:grid-cols-3">
+        {charts.map(({ data, title, type }, index) => (
```

Comment:
each of these charts should become its own component managing its own api hooks and relevant query params